CardForm/src/main/java/com/braintreepayments/cardform/utils/VibrationHelper.java

```diff
@@ -1,12 +1,14 @@
 package com.braintreepayments.cardform.utils;
 
 import android.Manifest;
+import android.annotation.SuppressLint;
 import android.content.Context;
 import android.content.pm.PackageManager;
 import android.os.Vibrator;
 
 public class VibrationHelper {
 
+    @SuppressLint("MissingPermission")
```

Comment:
We should investigate why this is necessary now when it wasn't before.

Reply:
I'm not to sure why the change to SDK 28 caused this lint error. This is an acceptable lint error to suppress though. We don't want to force the vibrate permission for the consuming app, but utilize it if it is available.